pages/linux/loginctl.md

```diff
@@ -22,3 +22,7 @@
 - Execute a `loginctl` operation on a remote host:
 
 `loginctl list-users -H {{hostname}}`
+
+- Log a user out on all of its sessions:
+
+`loginctl terminate-user {{username}}`
```

Comment:
Should we have `sudo` here?

```suggestion
`sudo loginctl terminate-user {{username}}`
```

The reason I'm not sure is, despite the fact that this command requires privilege escalation, it actually works without `sudo` too by authenticating using polkit.

Current guidelines really doesn't make it clear whether `sudo` should be included in such a case. I think it depends on how we interpret its purpose: is it here to

1. make the command work, or
2. indicate privilege escalation?

Personally I favour the second interpretation so I prefer adding `sudo`. I'd like to hear others' opinions.

Reply:
Loginctl without sudo applies only to the current user as in you can log yourself out of all of your instances. If you do not povide sudo and attempt to use it on someone else, the tool will ask for sudo password anyways.

Reply:
> Thus I don't see that adding sudo is necessary.

+1

Reply:
Systemctl acts the same way and its page doesn't use sudo.